Packs/Armis/Integrations/Armis/Armis.py

```diff
@@ -0,0 +1,669 @@
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    aql_string = args.get('aql_string')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.free_string_search_devices(aql_string, max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchDevicesByAql',
+            outputs_key_field='deviceId',
+            outputs=results,
+            raw_response=response,
+            readable_output=tableToMarkdown('Devices', results, headers=[
+                'riskLevel',
+                'name',
+                'dataSources',
+                'category',
+                'type',
+                'model',
+                'manufacturer',
+                'macAddress',
+                'ipAddress',
+                'site',
+                'tags',
+                'operatingSystem',
+                'operatingSystemVersion',
+                'sensor',
+                'user',
+                'accessSwitch',
+                'firstSeen',
+                'id',
+                'ipv6',
+                'lastSeen',
+                'visibility',
+            ], removeNull=True)
+        )
+    return 'No devices found'
+
+
+def search_alerts_by_aql_command(client: Client, args):
+    """
+   armis-search-alerts-by-aql command: Returns results for searching Alerts using a free AQL string
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    aql_string = args.get('aql_string')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.free_string_search_alerts(aql_string, max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchAlertsByAql',
+            outputs_key_field='alertId',
+            outputs=results,
+            raw_response=response,
+            readable_output=tableToMarkdown('Alerts', results, headers=[
+                'severity',
+                'type',
+                'time',
+                'status',
+                'title',
+                'description',
+                'activityIds',
+                'activityUUIDs',
+                'alertId',
+                'connectionIds',
+                'deviceIds',
+            ], removeNull=True)
+        )
+    return 'No alerts found'
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    secret = demisto.params().get('secret')
+
+    # get the service API url
+    base_url = demisto.params().get('url')
+    verify = not demisto.params().get('insecure', False)
+
+    # How much time before the first fetch to retrieve incidents
+    first_fetch_time = demisto.params().get('fetch_time', '3 days').strip()
+

```

Comment:
```suggestion
    params = demisto.params()
    secret = params.get('secret')

    # get the service API url
    base_url = params.get('url')
    verify = not params.get('insecure', False)

    # How much time before the first fetch to retrieve incidents
    first_fetch_time = params.get('fetch_time', '3 days').strip()

    proxy = params.get('proxy', False)

```

Please pay attention to my addition of the proxy parameter - this is very important for clients that are using proxy settings